packages/react-components/react-progress/package.json

```diff
@@ -48,8 +48,7 @@
   "beachball": {
     "disallowedChangeTypes": [
       "major",
-      "minor",
-      "patch"
+      "prerelease"
```

Comment:
You have a `Status: Alpha` warning in your README.md that also needs to be removed.

Reply:
This has now been removed from what I can see. Resolving this conversation.